lib/utapi/utilities.js

```diff
@@ -82,11 +98,15 @@ export function listBucketMetrics() {
         .option('-v, --verbose')
         .parse(process.argv);
 
-    const { host, port, accessKey, secretKey, start, end,
-        buckets, verbose, ssl } =
+    const { host, port, accessKey, secretKey, start, end, verbose,
+        ssl } =
         commander;
-    const requiredOptions = { host, port, accessKey, secretKey, buckets,
-        start };
+    const requiredOptions = { host, port, accessKey, secretKey };
+
+    // Ensure the proper metric is provided.
+    const metric = metricType === undefined ? commander.metric : metricType;
```

Comment:
maybe add comment like: if old style bucket metrics, metricType will be "buckets", otherwise, commander.metric should be defined.